File: src/IndoorCricket/Models/Team.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace IndoorCricket.Models
{
    public class Team
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }
        public string Name { get; set; }

        public virtual ICollection<Player> Players { get; set; }
    }

    public class Player
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }

        public virtual Team team { get; set; }
    }
}

File: src/IndoorCricket/wwwroot/app/models/Team.js
var Models;
(function (Models) {
    'use strict';
    var Team = (function () {
        function Team(team) {
            this.id = team.Id;
            this.name = team.Name;
        }
        return Team;
    })();
    Models.Team = Team;
})(Models || (Models = {}));
